src/modules/Dropdown/Dropdown.js

```diff
@@ -180,7 +180,7 @@ export default class Dropdown extends Component {
 
   componentWillReceiveProps(nextProps) {
     super.componentWillReceiveProps(nextProps)
-    debug.groupCollapsed('componentWillReceiveProps()')
+    debug('componentWillReceiveProps()')
```

Comment:
Group calls were converted to regular debug calls, so we still get the method logs.